docs/whats-new.md

```diff
@@ -4,26 +4,42 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
 
 ## deck.gl v7.3
 
-> TODO: GIF/screenshot - Spring transition, 3d tiles, Jupyter widget
+Release Date: Sep xx, 2019
 
-### Transitions
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/.gif" />
+        <p><i>Tile3DLayer</i></p>
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/.jpg" />
+        <p><i>Jupyter Widget</i></p>
+      </td>
+      <td>
+        <img style="max-height:200px" src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/.jpg" />
+        <p><i>Spring Transition</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
 
-- Generic prop transition: the layer `transitions` prop now supports many more props than just accessors! Any prop of type `number` or `array` can now also use the built-in transition system.
-- New transition type: spring-based transition support is added to the transition settings. See [documentation](/docs/api-reference/layer.md#transitions-object-optional) for details.
 
 ### Tile3DLayer
 
-deck.gl has partnered with [Cesium](https://cesium.com/) to implement support for the [3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification). This makes it possible to dynamically load and render infinitely large datasets into the browser.
+deck.gl has partnered with [Cesium](https://cesium.com/) to implement support for the OGC [3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification). This makes it possible to dynamically load and render infinitely large datasets into the browser.
 
 In this initial release, the layer has full support for point clouds and experimental support for glTF tiles. Try the [demo]() for yourself.
 
-### Built-in Tooltip
+### Python Binding and Jupyter Widget
```

Comment:
"Python Binding and Jupyter Widget" => 

"pydeck: deck.gl for Python and Jupyter Notebooks"